tests/builder/custom_params_test.py

```diff
@@ -143,7 +143,7 @@ def test_set_codePageRanges(self):
         self.assertEqual(self.ufo.info.openTypeOS2CodePageRanges, [0, 1])
 
     def test_set_openTypeOS2CodePageRanges(self):
-        self.font.customParameters['openTypeOS2CodePageRanges'] = [0, 1]
+        self.font.customParameters['openTypeOS2CodePageRanges'] = [1252, 1250]
```

Comment:
Can you add a test that round-trips one of the "unsupported bits"? E.g. you put the bit in UFO, go to Glyphs, check that there is "codePageRangesUnsupportedBits", then go back to UFO and check that it's still here?